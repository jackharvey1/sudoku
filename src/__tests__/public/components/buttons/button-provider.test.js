import React from 'react';
import ButtonProvider from '../../../../public/components/buttons/button-provider';
import Renderer from 'react-test-renderer';

describe('Button provider', () => {
    it('renders as intended', () => {
        const testRenderer = Renderer.create(
            <ButtonProvider
                onClick={() => null}
                icon={() => null}
                text={"Test"}
            />
        );
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
});
