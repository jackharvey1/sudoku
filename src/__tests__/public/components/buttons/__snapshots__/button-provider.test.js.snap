// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button provider renders as intended 1`] = `
<button
  className="css-mhex0s"
  onClick={[Function]}
>
  <div
    className="css-1sginf6"
  >
    <p>
      Test
    </p>
  </div>
</button>
`;
